### ISSUE TITLE
Error reporting on wrong usage of the CLI

### ISSUE THREAD

**Issue by awvwgk** [AstraZeneca/kallisto#29]
**Is your feature request related to a problem? Please describe.**
I just run `kallisto` over a set of structures with
```
for i in *; do
  kallisto alp $i/inp.xyz > $i/kallisto.out
done
```
But remembered a moment too late that `kallisto` expects the input file as argument to `--inp` and the positional argument is the output. Fortunately, the geometries were under version control, which made reverting easy.

At least to me the CLI feels from time to time unintuititive and I have to frequently consult the help printout to get the command line right.

**Describe the solution you'd like**
Print the error message to the stderr instead of the output file.

**Describe alternatives you've considered**
Adjust the CLI to expect the input file as positional argument and write to stdout.

**Comment by f3rmion** [comment]
Hi @awvwgk,

Good suggestion!

It is now implemented (230d91b) such that the input is used as a positional argument. 

```bash
> kallisto alp <input>
```

The output is now passed as an option instead.

```bash
> kallisto alp --out <output> <input>
```

I hope this makes the application of the CLI more intuitive.
Errors that are caught by the ``errorbye`` function are furthermore redirected to stderr instead of stdout.

**Comment by RMeli** [comment]
Great! I agree these changes are more in line with user's expectations. =)
